container/libcontainer/helpers.go

```diff
@@ -398,8 +414,8 @@ func scanUdpStats(r io.Reader) (info.UdpStat, error) {
 	return stats, nil
 }
 
-func GetProcesses(cgroupManager cgroups.Manager) ([]int, error) {
-	pids, err := cgroupManager.GetPids()
+func (l *Handler) GetProcesses() ([]int, error) {
```

Comment:
Maybe move all methods of the Handler struct together? Right now they are so scattered throughout the file. Maybe even move them out of `helpers.go`?

Reply:
done